CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 18.2.2 - [#742](https://github.com/openfisca/openfisca-france/pull/742)
```

Comment:
However, on a second thoughts, improving the error is a proper nice feature, so bumping the version number is legit.